### PR TITLE
Changes based on issue #97

### PR DESCRIPTION
**Pull request by rykal-nate**
Also a fix to throw identity exception when an identity does not exist in wallet immediately rather than attempting to build a gateway connection. 

### Checklist:

- [x] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [x] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [x] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure).

### Modifying an existing stack:

- [ ] Updated the stack version in `stack.yaml`

### Contributing a new stack:

- Describe how application dependencies are managed:

- Explain how Appsody file watcher is utilized:

- Describe other Appsody environment variables defined by the stack image:

- Describe any limitations and known issues:

### CODE REVIEW

File: incubator/blockchain-client-openliberty/templates/default/src/test/java/application/api/IdentityMapperFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        ArgumentCaptor<Response> argumentCaptor = ArgumentCaptor.forClass(Response.class);
+        filter.filter(reqContext);
+        verify(reqContext).abortWith(argumentCaptor.capture());
+
+        Response response = argumentCaptor.getValue();
+        System.out.println(response);
+    }
+
+    @Test
+    public void testFilterNullPrinciple() throws Exception {
```
Comment (chilanti) [inline incubator/blockchain-client-openliberty/templates/default/src/test/java/application/api/IdentityMapperFilterTest.java]: should be `testFilterNullPrincipal`.

**Review by chilanti — CHANGES_REQUESTED** [review]
LGTM - there's a "typo" in one of the test methods (testNullPrinciple instead of Principal).

File: incubator/blockchain-client-openliberty/templates/default/src/test/java/application/api/IdentityMapperFilterTest.java (hunk 1 above)
Comment (rykal-nate) [reply incubator/blockchain-client-openliberty/templates/default/src/test/java/application/api/IdentityMapperFilterTest.java]: Good catch - fixed committed to this branch.

**Review by chilanti — APPROVED** [review]
LGTM

**Pull request merged**